data/todo.py

```
@@ -1,24 +1,66 @@
+            .eq("id", todo_id) \
+            .eq("user_id", user_id) \
+            .execute()
+        
+        if response.data and len(response.data) > 0:
+            return response.data[0]
+        else:
+            print(f"[ERROR] No todo found or updated for todo_id {todo_id}, user_id {user_id}")
+            return None
+    except Exception as e:
+        print(f"[ERROR] Failed to update todo status for todo_id {todo_id}, user_id {user_id}: {e}")
+        raise e
```

Ruff (0.14.7)

Use `raise` without specifying exception name

Remove exception name

(TRY201)